src/lib/signature_lib/find_address/find_address.ml

```diff
@@ -132,7 +132,7 @@ let min_value_compressed = Public_key.Compressed.to_base58_check min_value
 let max_value : Public_key.Compressed.t =
   { x =
       Kimchi_backend.Pasta.Basic.Bigint256.of_hex_string
-        "0x0FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
+        "0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF0F"
```

Comment:
this should be reviewed by someone who knows that part of the code. I find it weird that we're encoding the maximum size of a public key here like that, if we change field this might not be valid anymore. IMO we should get the relevant field to return the maximum value instead (but I haven't spent enough time looking at this code to understand if my intuition is right).